### DISCUSSION TITLE
Audio file too large to be uploaded

### DISCUSSION THREAD

**Discussion by KOIexe86** [cc-tweaked/CC-Tweaked#1818] [Q&A] [answered]
Hi I want to play music in my hous with cc but when import the file (in DFPWM) it say that the file is too large to import.
I play on a server. Can it be fixed ?
Thanks

**Comment by Lupus590** [comment]
You can still upload the file to a file sharing website and then have the CC computer download/stream it.

**Reply by KOIexe86** [reply]
How do i do that ?

**Reply by Lupus590** [reply, marked as answer]
Upload your file to a service like Google Drive or DropBox.
Get a raw link to the file. This link should be the same one that would start the download in your web browser.
On the CC computer, use this link with the wget program to download the file.

**Reply by KOIexe86** [reply]
Thanks I will test when I can
